### PR TITLE
fix strategy of AsciiDocAPI Search for the asciidoc command file:

### PR DESCRIPTION
**Pull request by jujumo**
also search for sibling files before for system wide path.

### CODE REVIEW

**Comment by elextr** [comment]
Havn't tested but looks ok to me, @MasterOdin any comments?

@jujumo whilst I normally like a good argument about coding styles and especially something as contentious as "else considered harmful" for this just keeping the same style as the rest of the project is more important.  Thanks for your understanding.  BTW don't make a new PR for each change, just push correcting commits, we can squash them on merge.

If its suggested that the whole project style is to be changed then a separate issue just addressing that should be raised so that it can be discussed first, and (if its decided to go ahead) a suitable implementation strategy formulated (probably for progressive small changes).

**Comment by jujumo** [comment]
Thanks for your review and your patience. that is my first PR. Next time, I'll manage to make correcting commits.

File: asciidocapi.py [hunk 1]
```diff
@@ -189,16 +192,23 @@ def __init__(self, asciidoc_py=None):
             if not os.path.isfile(cmd):
                 raise AsciiDocError('missing file: %s' % cmd)
         else:
-            # Try shell search paths.
+            # try to find sibling
+            this_path = os.path.dirname(os.path.realpath(__file__))
+            # Try sibling paths.
```
Comment (MasterOdin) [inline asciidocapi.py]: I'd replace the comment two lines up with this one and just have one comment describing this block. Having these both with a line between them is unnecessary as they duplicate the content.

Comment (aerostitch) [inline asciidocapi.py]:
```suggestion
```

File: asciidocapi.py [hunk 2]
```diff
@@ -189,16 +192,23 @@ def __init__(self, asciidoc_py=None):
             if not os.path.isfile(cmd):
                 raise AsciiDocError('missing file: %s' % cmd)
         else:
-            # Try shell search paths.
+            # try to find sibling
```
Comment (aerostitch) [inline asciidocapi.py]:
```suggestion
            # try sibling paths.
```

**Pull request merged**